oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/util.js

```diff
@@ -45,12 +60,90 @@ export function createFormAppInit({
   autoInit = true,
   ContainerComponent = React.Fragment,
   componentOverrides = {},
+  recordSerializerProp,
+  recordApiClientProp,
+  fileApiClientProp,
+  draftsServiceProp,
+  configureStoreProp,
+  filesServiceProp,
+  depositServiceProp,
+  additionalApiConfigProp,
+  internalKeysProp,
+  keysToRemoveProp,
+  depositReducer = null,
+  filesReducer = null,
 } = {}) {
   const initFormApp = async ({ rootEl, ...config }) => {
     console.debug("Initializing Formik form app...");
     console.debug({ ...config });
     const overridableIdPrefix = config.formConfig.overridableIdPrefix;
 
+    const recordSerializer =
+      recordSerializerProp ||
+      new OARepoDepositSerializer(
+        internalKeysProp || ["errors", "expanded"],
+        keysToRemoveProp || ["__key"]
+      );
+
+    const additionalApiConfig = additionalApiConfigProp || {
```

Comment:
This can be imported/reused, shouldn't be defined here - constant of DepositApiClient base class